index.html

```diff
@@ -69,6 +70,10 @@ <h5 id="totalDuration">00:00:00</h5>
                         <div id="share" class="col s3 center waves-effect btn-play-control">
                             <i class="material-icons small" title="Copy Share Link">share</i>
                         </div>
+                        <div id="clear_playlist" class="col s2 center waves-effect btn-play-control">
+                            <i class="material-icons small" title="Clear Playlist">clear</i>
+                        </div>
```

Comment:
The different sized columns look weird currently. I have a suggestion, why don't you create an extended options menu which will dropdown (using [more_vert icon](https://material.io/icons/#ic_more_vert) ) and show both "Share" and "Clear playlist". That will look less cluttered.

You can copy the dropdown implementation of `play_mode`.

Reply:
Okay. I will create a extended icon